renderer/views/preferences.js

```diff
@@ -40,6 +42,27 @@ function renderDownloadDirSelector (state) {
   })
 }
 
+function renderExternalPlayerSelector (state) {
+  return renderFileSelector({
+    label: 'External Media Player',
+    description: 'Progam that will be used to play media externally',
+    property: 'externalPlayerPath',
+    options: {
+      title: 'Select media player executable',
+      properties: [ 'openFile' ]
+    }
+  },
+  state.unsaved.prefs.externalPlayerPath || '<VLC>', // TODO: should we get/store vlc path instead?
```

Comment:
It may be confusing to people that they can't set the path back to `<VLC>` after using a different player